### PR TITLE
Create GreekBERT_IB1S.py

### PR DESCRIPTION
**Pull request by alexandrosXe**
added greek bert

### CODE REVIEW

File: GreekBERT_IB1S.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import tensorflow as tf
+#from tokenizers import BertWordPieceTokenizer
+from transformers import BertTokenizer, TFBertModel, BertConfig
+from tqdm import tqdm
+
+
+class GreekBERT_IB1S():
```
Comment (ipavlopoulos) [inline GreekBERT_IB1S.py:6]: Add some class-level comments.

File: GreekBERT_IB1S.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import tensorflow as tf
+#from tokenizers import BertWordPieceTokenizer
```
Comment (ipavlopoulos) [inline GreekBERT_IB1S.py]: Remove?

File: GreekBERT_IB1S.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        self.monitor_mode = monitor_mode
+        self.dense_activation = dense_activation
+        self.earlystop = tf.keras.callbacks.EarlyStopping(monitor=self.monitor_loss,
+                                                            patience=self.patience,
+                                                            verbose=1,
+                                                            restore_best_weights=True,
+                                                            mode=self.monitor_mode)
+        self.BERT = TFBertModel.from_pretrained("nlpaueb/bert-base-greek-uncased-v1", output_attentions = True)
+        
+  #prepare inputs for bert
```
Comment (ipavlopoulos) [inline GreekBERT_IB1S.py]: Change to method-level comment.

File: GreekBERT_IB1S.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    for layer in self.BERT.layers:
+      for l in layer.encoder.layer:
+        if counter == len(layer.encoder.layer)-3:
+          return
+        else:
+          counter+=1
+          l.trainable = False
+
+  def build(self, bias=0):
+        #encode text via bert
```
Comment (ipavlopoulos) [inline GreekBERT_IB1S.py:73]: Ditto

**Review by ipavlopoulos — APPROVED** [review]

File: GreekBERT_IB1S.py (hunk 2 above)
Comment (alexandrosXe) [reply GreekBERT_IB1S.py]: fixed @ipavlopoulos

File: GreekBERT_IB1S.py (hunk 3 above)
Comment (alexandrosXe) [reply GreekBERT_IB1S.py]: @fixed

Comment (alexandrosXe) [reply GreekBERT_IB1S.py]: @ipavlopoulos

File: GreekBERT_IB1S.py (hunk 4 above)
Comment (alexandrosXe) [reply GreekBERT_IB1S.py:73]: i think this is ok @ipavlopoulos

Comment (ipavlopoulos) [reply GreekBERT_IB1S.py:73]: Follow PEP8 for documentation.

File: GreekBERT_IB1S.py [hunk 5]
```diff
@@ -44,11 +44,12 @@ def __init__(self,
...
-  '''
-  This method returns the inputs that Bert needs: input_ids, input_masks, input_segments 
-  '''
+        
   def to_bert_input(self, texts):
+    """ This method returns the inputs that Bert needs: input_ids, input_masks, input_segments 
+      :param texts: the texts that will be converted into bert inputs 
+      :return: a tuple containing the input_ids, input_masks, input_segments 
+    """
       input_ids, input_masks, input_segments = [],[],[]
```
Comment (ipavlopoulos) [inline GreekBERT_IB1S.py:53]: Looks good, please add a similar one to more methods.

**Review by ipavlopoulos — COMMENTED** [review]
Change inline comments (build, predict, etc.) to proper ones for methods.

**Pull request merged**